src/components/chat/ChatLine.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import { routes } from '../../constants/routesPaths';
+import * as constraints from '../../utils/constraints';
+import littleSmilies from '../../assets/little-smilies.svg';
+
+const ChatLine = ({ name, image, comment, topic }) => (
+  <div>
+    <div className="left">
+      <img src={littleSmilies}   className="bitmap"/>
```

Comment:
I think you have an extra space between `{littleSmilies}` and `className`. The same applies to a couple of lines below